src/lib/firebaseBackend.js

```diff
@@ -452,17 +452,28 @@ class FirebaseBackend extends BackendInterface {
 
   // VALIDATED DOMAINS
 
-  async getDomains(pendingOnly) {
+  async getDomains(pendingOnly, callback) {
```

Comment:
@newhouseb I tried to return the data (`newDomains`) with this function but it wouldn't trigger renders in the component so ended up passing a callback which passes the data.